lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -4500,7 +4500,7 @@ mockProtocolParametersForBalancing =
                 (577 % 10_000)
         , Cardano.protocolParamMaxBlockExUnits =
             Just $ Cardano.ExecutionUnits 10_000_000_000 14_000_000
-        , Cardano.protocolParamCollateralPercent = Just 1
+        , Cardano.protocolParamCollateralPercent = Just 150
```

Comment:
Just curious: why was this change necessary? (I'm assuming you had a good reason!)